tests/reactive_fluid_transport_no_reaction.prm

```diff
@@ -0,0 +1,144 @@
+set Nonlinear solver tolerance             = 1e-5
+
+# The number of space dimensions you want to run this program in.
+set Dimension                              = 2
+
+set End time                               = 10e3
+
+# As there are no reactions between the fluid and solid,
+# there is no need to use operator splitting.
+set Use operator splitting                     = false
+
+# Mellt transport is turned on to ensure it works
+# with this reaction scheme. If set to false,
+# a separate velocity and pressure would not be
+# calculated for the fluid phase, and the porosity
+# field would be advected with the solid velocity
+# unless a separate advection scheme is specified.
+subsection Melt settings
+  set Include melt transport                  = true
+end
+
+# There are two compositional fields, one that tracks the amount of free fluid
+# (the porosity) and one that tracks the amount of bound fluid (bound_fluid).
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = porosity, bound_fluid
+end
+
+# Initially, there is no free water (the porosity is zero) and the water content
+# of the material is 1%.
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Function expression = 0; 0.01
+  end
+end
+
+# The composition is not fixed on any boundary
+subsection Boundary composition model
+  set Fixed composition boundary indicators =
+  set List of model names = initial composition
+end
+
+# Free slip on all sides
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left, right, bottom, top
+end
+
+# The model is 2x2 km
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent  = 2e3
+    set Y extent  = 2e3
+  end
+
+end
+
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 10.0
+  end
+
+end
+
+# The temperature is not important for this model, so we simply
+# set it to 1600 K everywhere (including the boundary).
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 1600
+  end
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = top, bottom
+  set List of model names = initial temperature
+
+  subsection Initial temperature
+    set Maximal temperature = 1600
+    set Minimal temperature = 1600
+  end
+end
+
+
+# The reactive fluid transport model is composited
+# on top of the visco plastic material model.
+subsection Material model
+  set Model name = reactive fluid transport
+  subsection Reactive Fluid Transport Model
+    set Base model = visco plastic
+    set Reference fluid density = 2995
+    set Shear to bulk viscosity ratio = 0.1
+    set Reference fluid viscosity = 10
+    set Reference permeability = 2.5e-6
+    set Exponential fluid weakening factor = 0
+    set Fluid-solid reaction scheme = zero solubility
```

Comment:
Shouldn't this be no reaction? 
You switch the operator splitting off, so technically, this should give you the correct result, but I find it to be confusing.

Reply:
I see it actually does not work because
`Exception 'ExcMessage("The Fluid-reaction scheme zero solubility must be used with operator splitting.")'`
:-)